.gitignore

```diff
@@ -4,3 +4,13 @@
 build
 out
 gradle-local.properties
+
+bin/test/logback.xml
```

Comment:
Just ignore the entire "bin" directory, that'll be helpful for people whose tool defaults to compiling files under "bin" (such as VSCode)